TedTalks/TedTalks/Controllers/TedTalkDetailViewController.swift

```diff
@@ -17,7 +17,7 @@ class TedTalkDetailViewController: UIViewController {
     @IBOutlet weak var talkWebView: WKWebView!
     @IBOutlet weak var viewsLabel: UILabel!
     @IBOutlet weak var uploadDateLabel: UILabel!
-    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var descriptionTextView: UITextView!
```

Comment:
This text view shouldn't be editable.

Reply:
I don't understand this change. I mean, it needs to be editable as its text it's set in line 44: descriptionTextView.text = "Description: \(tedTalk.description)"

Maybe I'm getting you wrong

Reply:
Yeah, so TextViews can edit text so if you leave it as editable users will be able to edit whatever you put in there, in this case the description, you can do it in Interface Builder or in code

Reply:
Ohh, Understood, Editable unchecked.